storage: Lease owned by replica that no longer exists

The following tests appear to have failed:

[#226401](https://teamcity.cockroachdb.com/viewLog.html?buildId=226401):

```
--- FAIL: testrace/TestScatter (0.000s)
Test ended in panic.

------- Stdout: -------
F170417 20:12:25.169592 433090 storage/replica.go:1017  [n3,s3,r19/3:/Table/51/1/{70-80},@c4239c4600] lease repl={1 1 1} start=1492459945.035100453,0 epo=1 pro=1492459945.035111753,0 owned by replica {NodeID:1 StoreID:1 ReplicaID:1} that no longer exists
goroutine 433090 [running]:
github.com/cockroachdb/cockroach/pkg/util/log.getStacks(0x4100a01, 0xa1bc6d6, 0x3738d80, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:837 +0xc4
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).outputLogEntry(0x373a360, 0xc400000004, 0x2f7a53d, 0x12, 0x3f9, 0xc429ee5e10, 0xc6)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:709 +0x660
github.com/cockroachdb/cockroach/pkg/util/log.addStructured(0x7f170803d918, 0xc423337f80, 0x4, 0x2, 0x24739b6, 0x33, 0xc42a6a3ee8, 0x2, 0x2)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/structured.go:146 +0x313
github.com/cockroachdb/cockroach/pkg/util/log.logDepth(0x7f170803d918, 0xc423337f80, 0x1, 0x4, 0x24739b6, 0x33, 0xc42a6a3ee8, 0x2, 0x2)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:67 +0x9a
github.com/cockroachdb/cockroach/pkg/util/log.Fatalf(0x7f170803d918, 0xc423337f80, 0x24739b6, 0x33, 0xc42a6a3ee8, 0x2, 0x2)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/log.go:169 +0x90
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).redirectOnOrAcquireLease.func1(0xc4239c4600, 0xc42a6a42d0, 0x14b64869a947af6c, 0x0, 0x7f170803d918, 0xc423337f80, 0xc42a6a42b0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1017 +0xe3a
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).redirectOnOrAcquireLease(0xc4239c4600, 0x7f170803d918, 0xc423337f80, 0x0, 0x0, 0x0, 0x0, 0x0, 0x3)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1083 +0x1ed
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).tryExecuteWriteBatch(0xc4239c4600, 0x7f170803d918, 0xc423337f80, 0x14b64869a7b5052d, 0x0, 0x300000003, 0x3, 0x13, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2235 +0x1217
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).executeWriteBatch(0xc4239c4600, 0x7f170803d918, 0xc423337f80, 0x14b64869a7b5052d, 0x0, 0x300000003, 0x3, 0x13, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2121 +0xa9
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).Send(0xc4239c4600, 0x7f170803d918, 0xc423337f80, 0x14b64869a7b5052d, 0x0, 0x300000003, 0x3, 0x13, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1449 +0x28e
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Send(0xc4227dc400, 0x7f170803d918, 0xc42c07e7b0, 0x14b64869a7b5052d, 0x0, 0x300000003, 0x3, 0x13, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2548 +0x966
github.com/cockroachdb/cockroach/pkg/storage.(*Stores).Send(0xc4236fed80, 0x7f170803d918, 0xc42c07e780, 0x0, 0x0, 0x300000003, 0x3, 0x13, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/stores.go:187 +0x24b
github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:843 +0x227
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr(0xc42cab06e0, 0xc42a6a57e0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:263 +0x125
github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal(0xc4251238c0, 0x7f170803d918, 0xc42c07e750, 0xc42173d108, 0xc42c07e750, 0x13c0100, 0x244be1b)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:854 +0x1ca
github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch(0xc4251238c0, 0x7f170803d918, 0xc42c07e750, 0xc42173d108, 0x1f, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:871 +0xb8
github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext.func1.1(0xc4235ea2d0, 0xc42173d0e0, 0x7f170803d918, 0xc42c07e6f0, 0x128a5cd, 0x7f170803d918, 0xc4238224b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:192 +0x73d
github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext.func1(0xc4235ea2d0, 0xc42173d0e0, 0x7f170803d918, 0xc42c07e6f0, 0x330e700, 0x4100d68, 0xc423a41c80)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:205 +0xce
created by github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:209 +0x224

goroutine 1 [chan receive]:
testing.(*T).Run(0xc420194410, 0x242ab2c, 0xb, 0x24a0658, 0xc4204b5b01)
	/usr/local/go/src/testing/testing.go:698 +0x582
testing.runTests.func1(0xc420194410)
	/usr/local/go/src/testing/testing.go:882 +0xab
testing.tRunner(0xc420194410, 0xc4204b5d10)
	/usr/local/go/src/testing/testing.go:657 +0x108
testing.runTests(0xc4202e8d20, 0x32e3e40, 0x88, 0x88, 0x2f82094)
	/usr/local/go/src/testing/testing.go:888 +0x4e1
testing.(*M).Run(0xc4202a7f28, 0x41007c0)
	/usr/local/go/src/testing/testing.go:822 +0x1c4
github.com/cockroachdb/cockroach/pkg/sql_test.TestMain(0xc4204b5f28)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/main_test.go:227 +0x18a
main.main()
	github.com/cockroachdb/cockroach/pkg/sql/_test/_testmain.go:340 +0x210

goroutine 17 [syscall, 8 minutes, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 5 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x373a360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1003 +0x85
created by github.com/cockroachdb/cockroach/pkg/util/log.init.1
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:566 +0x13a

goroutine 6 [syscall, 8 minutes]:
os/signal.signal_recv(0x6ff371)
	/usr/local/go/src/runtime/sigqueue.go:116 +0x104
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x30
created by os/signal.init.1
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x4f

goroutine 449 [select, locked to thread]:
runtime.gopark(0x24a2830, 0x0, 0x2423f07, 0x6, 0x18, 0x2)
	/usr/local/go/src/runtime/proc.go:271 +0xfd
runtime.selectgoImpl(0xc420040f50, 0xc420040f38, 0x0)
	/usr/local/go/src/runtime/select.go:423 +0x138a
runtime.selectgo(0xc420040f50)
	/usr/local/go/src/runtime/select.go:238 +0x1c
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:434 +0x1ea
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2197 +0x1

goroutine 430280 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc431f6c8e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429149 [semacquire]:
sync.runtime_notifyListWait(0xc42b63ccd0, 0xc40000088f)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc42b63ccc0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422288990, 0xc424512d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc421b7e9a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 433016 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).tryExecuteWriteBatch(0xc420527b00, 0x7f170803d918, 0xc421d20120, 0x14b64869a87f9e21, 0x0, 0x300000003, 0x3, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2286 +0xf48
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).executeWriteBatch(0xc420527b00, 0x7f170803d918, 0xc421d20120, 0x14b64869a87f9e21, 0x0, 0x300000003, 0x3, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:2121 +0xa9
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).Send(0xc420527b00, 0x7f170803d918, 0xc421d20120, 0x14b64869a87f9e21, 0x0, 0x300000003, 0x3, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1449 +0x28e
github.com/cockroachdb/cockroach/pkg/storage.(*Store).Send(0xc4227dc400, 0x7f170803d918, 0xc421d200f0, 0x14b64869a87f9e21, 0x0, 0x300000003, 0x3, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2548 +0x966
github.com/cockroachdb/cockroach/pkg/storage.(*Stores).Send(0xc4236fed80, 0x7f170803d918, 0xc421d200c0, 0x0, 0x0, 0x300000003, 0x3, 0x1, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/stores.go:187 +0x24b
github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal.func1(0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:843 +0x227
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunTaskWithErr(0xc42cab06e0, 0xc429d2d7e0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:263 +0x125
github.com/cockroachdb/cockroach/pkg/server.(*Node).batchInternal(0xc4251238c0, 0x7f170803d918, 0xc421d20090, 0xc420336fc8, 0xc421d20090, 0x13c0100, 0x244be1b)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:854 +0x1ca
github.com/cockroachdb/cockroach/pkg/server.(*Node).Batch(0xc4251238c0, 0x7f170803d918, 0xc421d20090, 0xc420336fc8, 0x1f, 0x58f521a9, 0x8c70893)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:871 +0xb8
github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext.func1.1(0xc4236f4cd0, 0xc420336fa0, 0x7f170803d918, 0xc421d20000, 0x128a5cd, 0x7f17080960f8, 0xc420014e28)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:192 +0x73d
github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext.func1(0xc4236f4cd0, 0xc420336fa0, 0x7f170803d918, 0xc421d20000, 0x330e700, 0x4100d68, 0xc425fb99e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:205 +0xce
created by github.com/cockroachdb/cockroach/pkg/kv.(*grpcTransport).SendNext
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/transport.go:209 +0x224

goroutine 429721 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42630ca50)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429167 [semacquire]:
sync.runtime_notifyListWait(0xc42b63ccd0, 0xc400000865)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc42b63ccc0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422288990, 0xc424512d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc421b7f460)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430114 [semacquire]:
sync.runtime_notifyListWait(0xc421f940d0, 0xc400000628)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc421f940c0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4282be000, 0xc42cab06e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089d00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429200 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).raftTickLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3465 +0x481
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc421ad9e00)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429643 [chan receive]:
github.com/cockroachdb/cockroach/pkg/rpc.NewContext.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:176 +0x95
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42665d140)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429920 [select]:
github.com/cockroachdb/cockroach/pkg/sql/distsqlrun.(*flowScheduler).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsqlrun/flow_scheduler.go:114 +0x61c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc4257f5f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430186 [IO wait]:
net.runtime_pollWait(0x7f16a6425ad8, 0x72, 0x72db5a)
	/usr/local/go/src/runtime/netpoll.go:164 +0x5e
net.(*pollDesc).wait(0xc42c507b18, 0x72, 0x32e8cc8, 0xffffffffffffffff)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x5b
net.(*pollDesc).waitRead(0xc42c507b18, 0xc423852000, 0x2000)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x42
net.(*netFD).Read(0xc42c507ab0, 0xc423852000, 0x2000, 0x2000, 0x0, 0x32f35a0, 0x32e8cc8)
	/usr/local/go/src/net/fd_unix.go:250 +0x22b
net.(*conn).Read(0xc4206c0120, 0xc423852000, 0x2000, 0x2000, 0x102, 0xc42385200d, 0x0)
	/usr/local/go/src/net/net.go:181 +0x97
net.(*TCPConn).Read(0xc4206c0120, 0xc423852000, 0x2000, 0x2000, 0xc42186f6e8, 0xa4b5d6, 0xc42186f6e8)
	<autogenerated>:72 +0x73
crypto/tls.(*block).readFromUntil(0xc42acd8ae0, 0x7f1708045708, 0xc4206c0120, 0x5, 0xc4206c0120, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:488 +0x114
crypto/tls.(*Conn).readRecord(0xc42271d880, 0x24a2917, 0xc42271d9a0, 0xc42385200d)
	/usr/local/go/src/crypto/tls/conn.go:590 +0x13d
crypto/tls.(*Conn).Read(0xc42271d880, 0xc42b546000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1134 +0x190
bufio.(*Reader).Read(0xc424d0f440, 0xc426d17c38, 0x9, 0x9, 0x71ed23, 0xc420221040, 0xffffffff)
	/usr/local/go/src/bufio/bufio.go:213 +0x7e7
io.ReadAtLeast(0x32ea9a0, 0xc424d0f440, 0xc426d17c38, 0x9, 0x9, 0x9, 0x7003e2, 0x6dbaac, 0xc421d57500)
	/usr/local/go/src/io/io.go:307 +0xb7
io.ReadFull(0x32ea9a0, 0xc424d0f440, 0xc426d17c38, 0x9, 0x9, 0xc423c9b860, 0xc423c9b860, 0xc42186fe98)
	/usr/local/go/src/io/io.go:325 +0x73
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc426d17c38, 0x9, 0x9, 0x32ea9a0, 0xc424d0f440, 0x0, 0x12d00000000, 0x12d, 0x12d)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0xa3
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc426d17c00, 0x32f5b20, 0xc423c9b830, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0x118
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc42acd9e30, 0xc423c9b830, 0xc423c9b800, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:508 +0x53
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc4235dbc80)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1043 +0x15b
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:251 +0x1294

goroutine 429245 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc4245ec320)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430289 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc431f6c970)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429235 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*SchemaChangeManager).Start.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/schema_changer.go:761 +0x1c61
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc426a5e920)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430082 [semacquire]:
sync.runtime_notifyListWait(0xc421f940d0, 0xc40000061a)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc421f940c0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4282be000, 0xc42cab06e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089840)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429166 [semacquire]:
sync.runtime_notifyListWait(0xc42b63ccd0, 0xc40000086b)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc42b63ccc0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422288990, 0xc424512d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc421b7f3a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429185 [semacquire]:
sync.runtime_notifyListWait(0xc42b63ccd0, 0xc400000869)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc42b63ccc0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc422288990, 0xc424512d20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc42ce820c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430008 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423770f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430002 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423770ec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430972 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc422992e00, 0xc423b9e670, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:143 +0x70a
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*Stream).Read(0xc422a18000, 0xc423b9e670, 0x5, 0x5, 0x1c425e0, 0x7f16c81f1488, 0xc4243b1608)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:333 +0x7f
io.ReadAtLeast(0x32ef860, 0xc422a18000, 0xc423b9e670, 0x5, 0x5, 0x5, 0xc422dbfb78, 0x6e0961, 0xc4230c89c0)
	/usr/local/go/src/io/io.go:307 +0xb7
io.ReadFull(0x32ef860, 0xc422a18000, 0xc423b9e670, 0x5, 0x5, 0x71e8b8, 0xc424bfb950, 0xc422dbfbf0)
	/usr/local/go/src/io/io.go:325 +0x73
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*parser).recvMsg(0xc423b9e660, 0x7fffffff, 0x6a, 0x6a, 0xc422dbfd48, 0x6cbf1b, 0x6b31ad, 0xc41fed3fad)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:243 +0xa0
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.recv(0xc423b9e660, 0x33025a0, 0x4100d68, 0xc422a18000, 0x32f76a0, 0x4100d68, 0x2352b00, 0xc422580a40, 0x7fffffff, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:339 +0x61
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*serverStream).RecvMsg(0xc4298c83c0, 0x2352b00, 0xc422580a40, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/stream.go:613 +0x1d5
github.com/cockroachdb/cockroach/pkg/storage.(*multiRaftRaftMessageBatchServer).Recv(0xc42b342800, 0x7f170803d918, 0xc4202fd2f0, 0xc42653f670)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft.pb.go:351 +0x86
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch.func1.1.1(0x330bc00, 0xc42b342800, 0xc421b81c80, 0xc422a18028, 0xc421208fa8)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:300 +0x61
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).RaftMessageBatch.func1.1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:323 +0x61
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc4202fd320)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430129 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x4b3
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430283 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc431f6c910)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429773 [semacquire]:
sync.runtime_notifyListWait(0xc424bfb950, 0xc400000500)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc424bfb940)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4243b15f0, 0xc420602aa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42395cf40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430292 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc431f6c9d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429663 [select]:
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc4243b14d0, 0xc42161d9c0, 0xc42a0329d0, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:356 +0xb5b
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.2.1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:297 +0x86
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc4229bd950)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430386 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*Store).startGossip.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:1355 +0x68b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc42c715b30)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 432189 [select]:
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendToReplicas(0xc428ae9100, 0x7f17080454f0, 0xc421a75e80, 0x1dcd6500, 0x0, 0xc428ae9148, 0x12, 0xc42af6a140, 0x3, 0x3, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:1165 +0x20cb
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendRPC(0xc428ae9100, 0x7f17080454f0, 0xc421a75e80, 0x12, 0xc42af6a140, 0x3, 0x3, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:398 +0x3b4
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendSingleRange(0xc428ae9100, 0x7f170803d918, 0xc42176c480, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:467 +0x1a8
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendPartialBatch(0xc428ae9100, 0x7f170803d918, 0xc42176c480, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:934 +0x51f
github.com/cockroachdb/cockroach/pkg/kv.(*DistSender).sendPartialBatchAsync.func1(0x7f170803d918, 0xc42176c480)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/dist_sender.go:869 +0x1c6
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask.func1(0xc424512d20, 0x2f7c0bc, 0x11, 0x367, 0xc4251cae40, 0x330e700, 0x4100d68, 0xc4269fc4e0, 0x7f170803d918, 0xc42176c480)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:348 +0x11a
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunLimitedAsyncTask
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:349 +0x2c2

goroutine 429521 [semacquire]:
sync.runtime_SemacquireMutex(0xc4256a9324)
	/usr/local/go/src/runtime/sema.go:62 +0x34
sync.(*Mutex).Lock(0xc4256a9320)
	/usr/local/go/src/sync/mutex.go:87 +0xda
sync.(*RWMutex).Lock(0xc4256a9320)
	/usr/local/go/src/sync/rwmutex.go:86 +0x49
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*RWMutex).Lock(0xc4256a9320)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:70 +0x3b
github.com/cockroachdb/cockroach/pkg/storage.(*Store).processRequestQueue(0xc4256a9000, 0x15)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3360 +0x65
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4243b15f0, 0xc420602aa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:225 +0x30c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42395c900)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429709 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42630c970)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430107 [runnable]:
sync.(*RWMutex).RUnlock(0xc42be598f0)
	/usr/local/go/src/sync/rwmutex.go:55 +0xe6
github.com/cockroachdb/cockroach/pkg/storage.(*Replica).Desc(0xc42be59800, 0xc4240108a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/replica.go:1179 +0xa8
github.com/cockroachdb/cockroach/pkg/storage.(*Store).tryGetOrCreateReplica(0xc4227dc400, 0x10, 0x5, 0xc4222257f8, 0x6cc16c, 0xc42bec54c0, 0xc42bec54c0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3622 +0x3bf
github.com/cockroachdb/cockroach/pkg/storage.(*Store).getOrCreateReplica(0xc4227dc400, 0x10, 0x5, 0xc4222257f8, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3594 +0x5f
github.com/cockroachdb/cockroach/pkg/storage.(*Store).processRaftRequest(0xc4227dc400, 0x7f170803d918, 0xc421f49320, 0xc4222257f0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:2859 +0xbc
github.com/cockroachdb/cockroach/pkg/storage.(*Store).processRequestQueue(0xc4227dc400, 0x10)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3368 +0x1f0
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4282be000, 0xc42cab06e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:225 +0x30c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089c20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430124 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x4b3
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089e80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429222 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc4245ec0e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430093 [semacquire]:
sync.runtime_notifyListWait(0xc421f940d0, 0xc400000621)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc421f940c0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4282be000, 0xc42cab06e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423089a20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429751 [semacquire]:
sync.runtime_notifyListWait(0xc424bfb950, 0xc4000004d7)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc424bfb940)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4243b15f0, 0xc420602aa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42395cc60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430021 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423771020)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429666 [IO wait]:
net.runtime_pollWait(0x7f170809f990, 0x72, 0x32e8cc8)
	/usr/local/go/src/runtime/netpoll.go:164 +0x5e
net.(*pollDesc).wait(0xc421fbdf78, 0x72, 0x32e8cc8, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x5b
net.(*pollDesc).waitRead(0xc421fbdf78, 0xffffffffffffffff, 0x0)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x42
net.(*netFD).accept(0xc421fbdf10, 0x0, 0x32f0820, 0xc422b5e460)
	/usr/local/go/src/net/fd_unix.go:430 +0x2c0
net.(*TCPListener).accept(0xc42c6ea810, 0xc42c413470, 0xc4212066f8, 0x6cb842)
	/usr/local/go/src/net/tcpsock_posix.go:136 +0x51
net.(*TCPListener).Accept(0xc42c6ea810, 0x24a0dd0, 0xc424bfb700, 0xc427d75b80, 0x6cbb1e)
	/usr/local/go/src/net/tcpsock.go:228 +0x50
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*cMux).Serve(0xc424bfb700, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:124 +0xdd
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func3()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:507 +0x47
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc4209440e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430395 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x4b3
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc4203adea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430176 [semacquire]:
sync.runtime_notifyListWait(0xc4278a0c10, 0xc4000003ab)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4278a0c00)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b5c1dd0, 0xc420ebd040)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc4203ad080)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429906 [select]:
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).runHeartbeat(0xc420344900, 0xc420c7a080, 0xc42333b1e0, 0xf, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:356 +0xb5b
github.com/cockroachdb/cockroach/pkg/rpc.(*Context).GRPCDial.func1.2.1()
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:297 +0x86
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc42c678390)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429929 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Node).startGossip.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:660 +0x64b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc423e8b760)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430215 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).controller(0xc423a18700)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:742 +0xa23
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Server
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:161 +0xdf2

goroutine 429866 [IO wait]:
net.runtime_pollWait(0x7f1687309198, 0x72, 0x72db5a)
	/usr/local/go/src/runtime/netpoll.go:164 +0x5e
net.(*pollDesc).wait(0xc422e15a38, 0x72, 0x32e8cc8, 0xffffffffffffffff)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x5b
net.(*pollDesc).waitRead(0xc422e15a38, 0xc426bcac00, 0x400)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x42
net.(*netFD).Read(0xc422e159d0, 0xc426bcac00, 0x400, 0x400, 0x0, 0x32f35a0, 0x32e8cc8)
	/usr/local/go/src/net/fd_unix.go:250 +0x22b
net.(*conn).Read(0xc4211ca440, 0xc426bcac00, 0x400, 0x400, 0x102, 0x6db864, 0x1c43ae0)
	/usr/local/go/src/net/net.go:181 +0x97
net.(*TCPConn).Read(0xc4211ca440, 0xc426bcac00, 0x400, 0x400, 0xc423ad56e8, 0xa4b5d6, 0xc423ad56e8)
	<autogenerated>:72 +0x73
crypto/tls.(*block).readFromUntil(0xc423b78240, 0x7f1708045708, 0xc4211ca440, 0x5, 0xc4211ca440, 0x6e50f0)
	/usr/local/go/src/crypto/tls/conn.go:488 +0x114
crypto/tls.(*Conn).readRecord(0xc4202ef500, 0x24a2917, 0xc4202ef620, 0x6b2f22)
	/usr/local/go/src/crypto/tls/conn.go:590 +0x13d
crypto/tls.(*Conn).Read(0xc4202ef500, 0xc420cc0000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1134 +0x190
bufio.(*Reader).Read(0xc4213b2780, 0xc425ad4038, 0x9, 0x9, 0x0, 0x0, 0xc423ad5be0)
	/usr/local/go/src/bufio/bufio.go:213 +0x7e7
io.ReadAtLeast(0x32ea9a0, 0xc4213b2780, 0xc425ad4038, 0x9, 0x9, 0x9, 0xc4207e11e0, 0xffffffff, 0xc420c21f40)
	/usr/local/go/src/io/io.go:307 +0xb7
io.ReadFull(0x32ea9a0, 0xc4213b2780, 0xc425ad4038, 0x9, 0x9, 0xd90268, 0xc424852130, 0x0)
	/usr/local/go/src/io/io.go:325 +0x73
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc425ad4038, 0x9, 0x9, 0x32ea9a0, 0xc4213b2780, 0x0, 0xc400000000, 0xc42c0e1d40, 0x32eb0a0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0xa3
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc425ad4000, 0x0, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0x118
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc42c8de1b0, 0xc421d20960, 0xc421d20900, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:508 +0x53
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Client).reader(0xc424852000)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:1043 +0x15b
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.newHTTP2Client
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_client.go:251 +0x1294

goroutine 429642 [select]:
github.com/cockroachdb/cockroach/pkg/security.(*CertificateManager).RegisterSignalHandler.func1(0xc420602aa0, 0xc42a648c60, 0xc4227838c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/security/certificate_manager.go:91 +0x511
created by github.com/cockroachdb/cockroach/pkg/security.(*CertificateManager).RegisterSignalHandler
	/go/src/github.com/cockroachdb/cockroach/pkg/security/certificate_manager.go:104 +0xf1

goroutine 430351 [semacquire]:
sync.runtime_notifyListWait(0xc4278a0c10, 0xc4000003ba)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4278a0c00)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b5c1dd0, 0xc420ebd040)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc4203ad920)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430568 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.newClientStream.func3(0x3309f80, 0xc4235dbc80, 0xc4273b39e0, 0xc4206286e0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/stream.go:236 +0x52e
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.newClientStream
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/stream.go:256 +0x1389

goroutine 430064 [semacquire]:
sync.runtime_notifyListWait(0xc421f940d0, 0xc40000063b)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc421f940c0)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc4282be000, 0xc42cab06e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc4230897c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429218 [select]:
github.com/cockroachdb/cockroach/pkg/server.(*Node).startWriteSummaries.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:738 +0x3bd
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc424512d20, 0xc4208f7dd0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430343 [semacquire]:
sync.runtime_notifyListWait(0xc4278a0c10, 0xc4000003d6)
	/usr/local/go/src/runtime/sema.go:298 +0x11a
sync.(*Cond).Wait(0xc4278a0c00)
	/usr/local/go/src/sync/cond.go:57 +0x97
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b5c1dd0, 0xc420ebd040)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:196 +0xe2
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:165 +0x4b
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc4203ad800)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429791 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*client).gossip(0xc4252ad2b0, 0x7f17080454f0, 0xc420c7a000, 0xc42706a480, 0x330ab80, 0xc422ae8b10, 0xc42cab06e0, 0xc42333a010, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/client.go:325 +0x4f9
github.com/cockroachdb/cockroach/pkg/gossip.(*client).startLocked.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/client.go:132 +0x6e5
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42cab06e0, 0xc4227b5940)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 429727 [select]:
github.com/cockroachdb/cockroach/pkg/sql.(*distSQLPlanner).initRunners.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:81 +0x1cb
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420602aa0, 0xc42630cab0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430984 [IO wait]:
net.runtime_pollWait(0x7f17080a0110, 0x72, 0x72db5a)
	/usr/local/go/src/runtime/netpoll.go:164 +0x5e
net.(*pollDesc).wait(0xc428da8308, 0x72, 0x32e8cc8, 0xffffffffffffffff)
	/usr/local/go/src/net/fd_poll_runtime.go:75 +0x5b
net.(*pollDesc).waitRead(0xc428da8308, 0xc4207e8000, 0x2000)
	/usr/local/go/src/net/fd_poll_runtime.go:80 +0x42
net.(*netFD).Read(0xc428da82a0, 0xc4207e8000, 0x2000, 0x2000, 0x0, 0x32f35a0, 0x32e8cc8)
	/usr/local/go/src/net/fd_unix.go:250 +0x22b
net.(*conn).Read(0xc420228a00, 0xc4207e8000, 0x2000, 0x2000, 0x555, 0x0, 0x24a2720)
	/usr/local/go/src/net/net.go:181 +0x97
net.(*TCPConn).Read(0xc420228a00, 0xc4207e8000, 0x2000, 0x2000, 0x1, 0x32eb0a0, 0xc420014110)
	<autogenerated>:72 +0x73
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*MuxConn).Read(0xc42b107ea0, 0xc4207e8000, 0x2000, 0x2000, 0xc42b4e14d0, 0xa4b5d6, 0xc42b4e14d0)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:218 +0x183
crypto/tls.(*block).readFromUntil(0xc423c56750, 0x7f17042ff108, 0xc42b107ea0, 0x5, 0xc42b107ea0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:488 +0x114
crypto/tls.(*Conn).readRecord(0xc429eb6700, 0x24a2917, 0xc429eb6820, 0xc4207e800d)
	/usr/local/go/src/crypto/tls/conn.go:590 +0x13d
crypto/tls.(*Conn).Read(0xc429eb6700, 0xc424336000, 0x8000, 0x8000, 0x0, 0x0, 0x0)
	/usr/local/go/src/crypto/tls/conn.go:1134 +0x190
bufio.(*Reader).Read(0xc4288912c0, 0xc422a13378, 0x9, 0x9, 0x7f16d82b34b8, 0xc424439268, 0x0)
	/usr/local/go/src/bufio/bufio.go:213 +0x7e7
io.ReadAtLeast(0x32ea9a0, 0xc4288912c0, 0xc422a13378, 0x9, 0x9, 0x9, 0xc42b4e1a48, 0x71ed23, 0xc420dc11e0)
	/usr/local/go/src/io/io.go:307 +0xb7
io.ReadFull(0x32ea9a0, 0xc4288912c0, 0xc422a13378, 0x9, 0x9, 0x0, 0x7f170819bc20, 0x7003e2)
	/usr/local/go/src/io/io.go:325 +0x73
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.readFrameHeader(0xc422a13378, 0x9, 0x9, 0x32ea9a0, 0xc4288912c0, 0x0, 0xc400000000, 0xc, 0x52f)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:237 +0xa3
github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2.(*Framer).ReadFrame(0xc422a13340, 0x32f5b20, 0xc42ca84450, 0x0, 0xc426c00100)
	/go/src/github.com/cockroachdb/cockroach/vendor/golang.org/x/net/http2/frame.go:492 +0x118
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*framer).readFrame(0xc424438ed0, 0xc42ca84450, 0xc42ca84450, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http_util.go:508 +0x53
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*http2Server).HandleStreams(0xc422a135e0, 0xc424439080, 0x24a1ae0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/http2_server.go:307 +0x3c4
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveStreams(0xc42355a540, 0x3309fe0, 0xc422a135e0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:505 +0x1e8
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).serveHTTP2Transport(0xc42355a540, 0x330ab20, 0xc429eb6700, 0x32f29a0, 0xc427064370)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:486 +0x5a6
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).handleRawConn(0xc42355a540, 0x330bd20, 0xc42b107ea0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:457 +0x5f4
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:426 +0x69c

goroutine 430180 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*addrConn).transportMonitor(0xc422f83a00)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:859 +0x7cf
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ClientConn).resetAddrConn.func1(0xc422f83a00)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:604 +0x267
created by github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*ClientConn).resetAddrConn
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/clientconn.go:605 +0xa4d

goroutine 430196 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc425c01bc0, 0xc42463ef00, 0x22590a0, 0x32cac10, 0x233a880)
	<autogenerated>:9 +0x9f
net/http.(*Server).Serve(0xc42afcbce0, 0x3301fa0, 0xc425c01bc0, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2643 +0x2a8
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func4()
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:511 +0x54
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc425c01c40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430391 [select]:
github.com/cockroachdb/cockroach/pkg/storage.(*baseQueue).processLoop.func1()
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/queue.go:490 +0x4b3
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc420ebd040, 0xc4203ade20)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:210 +0x8b
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:211 +0x74

goroutine 430494 [select]:
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*recvBufferReader).Read(0xc422ec1080, 0xc423a656b0, 0x5, 0x5, 0x0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:143 +0x70a
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport.(*Stream).Read(0xc4273b39e0, 0xc423a656b0, 0x5, 0x5, 0x6af9fa, 0xc42023a000, 0x30)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/transport/transport.go:333 +0x7f
io.ReadAtLeast(0x32ef860, 0xc4273b39e0, 0xc423a656b0, 0x5, 0x5, 0x5, 0x5090abf30819a4b0, 0xc421b59ac8, 0x6ac61b)
	/usr/local/go/src/io/io.go:307 +0xb7
io.ReadFull(0x32ef860, 0xc4273b39e0, 0xc423a656b0, 0x5, 0x5, 0x13e2599, 0xc421b59ac8, 0x8)
	/usr/local/go/src/io/io.go:325 +0x73
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*parser).recvMsg(0xc423a656a0, 0x7fffffff, 0xc421b59c10, 0x6cbf1b, 0xc42001d300, 0xc42001d300, 0x1, 0x6dbd32)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:243 +0xa0
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.recv(0xc423a656a0, 0x33025a0, 0x4100d68, 0xc4273b39e0, 0x32f76a0, 0x4100d68, 0x2352be0, 0xc4237d5980, 0x7fffffff, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/rpc_util.go:339 +0x61
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*clientStream).RecvMsg(0xc4206286e0, 0x2352be0, 0xc4237d5980, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/stream.go:386 +0x20e
github.com/cockroachdb/cockroach/pkg/storage.(*multiRaftRaftMessageBatchClient).Recv(0xc42a00efd0, 0x7f170803d918, 0xc4237d5920, 0xc4237d53b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft.pb.go:283 +0x86
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).processQueue.func1.1.1(0x330bba0, 0xc42a00efd0, 0xc422ec0dc0, 0xc422ebc600, 0x7f170803d918, 0xc424f48de0, 0xc42a761f98, 0x6cb842)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/raft_transport.go:467 +0x5a
github.com/cockroachdb/cockroach/pkg/storage.(*RaftTransport).processQueue.func1.1
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/roachpb/api.go
+++ b/pkg/roachpb/api.go
@@ -78,11 +78,6 @@ const (
 	isRange                // range commands may span multiple keys
 	isReverse              // reverse commands traverse ranges in descending direction
 	isAlone                // requests which must be alone in a batch
-	// Some commands can skip interacting with the command queue and the timestamp
-	// cache. For example, RequestLeaseRequest is sequenced exclusively by Raft.
-	// These requests still have keys in their header, but those keys are used
-	// exclusively for routing the request to the right range.
-	isNonKV
 	// Requests for acquiring a lease skip the (proposal-time) check that the
 	// proposing replica has a valid lease.
 	skipLeaseCheck
@@ -900,20 +895,16 @@ func (*RangeLookupRequest) flags() int         { return isRead }
 func (*ResolveIntentRequest) flags() int       { return isWrite }
 func (*ResolveIntentRangeRequest) flags() int  { return isWrite | isRange }
 func (*NoopRequest) flags() int                { return isRead } // slightly special
-func (*TruncateLogRequest) flags() int         { return isWrite | isNonKV }
-
-// MergeRequests are considered "non KV" because they do not need to be gated
-// by the command queue (reordering is ok) and they operate on non-MVCC data so
-// the timestamp cache is also unnecessary.
-func (*MergeRequest) flags() int { return isWrite | isNonKV }
+func (*TruncateLogRequest) flags() int         { return isWrite }
+func (*MergeRequest) flags() int               { return isWrite }
 
 func (*RequestLeaseRequest) flags() int {
-	return isWrite | isAlone | isNonKV | skipLeaseCheck
+	return isWrite | isAlone | skipLeaseCheck
 }
 
 // LeaseInfoRequest is usually executed in an INCONSISTENT batch, which has the
 // effect of the `skipLeaseCheck` flag that lease write operations have.
-func (*LeaseInfoRequest) flags() int { return isRead | isNonKV | isAlone }
+func (*LeaseInfoRequest) flags() int { return isRead | isAlone }
 func (*TransferLeaseRequest) flags() int {
 	// TODO(andrei): update this comment.
 	// TransferLeaseRequest requires the lease, which is checked in
@@ -924,9 +915,9 @@ func (*TransferLeaseRequest) flags() int {
 	// replica has registered that a transfer is in progress and
 	// `redirectOnOrAcquireLease` already tentatively redirects to the
 	// future lease holder.
-	return isWrite | isAlone | isNonKV | skipLeaseCheck
+	return isWrite | isAlone | skipLeaseCheck
 }
-func (*ComputeChecksumRequest) flags() int          { return isWrite | isNonKV | isRange }
+func (*ComputeChecksumRequest) flags() int          { return isWrite | isRange }
 func (*DeprecatedVerifyChecksumRequest) flags() int { return isWrite }
 func (*CheckConsistencyRequest) flags() int         { return isAdmin | isRange }
 func (*WriteBatchRequest) flags() int               { return isWrite | isRange }

--- a/pkg/roachpb/batch.go
+++ b/pkg/roachpb/batch.go
@@ -115,17 +115,6 @@ func (ba *BatchRequest) IsSingleRequest() bool {
 	return len(ba.Requests) == 1
 }
 
-// IsNonKV returns true iff all of the requests in the batch have the non-KV
-// flag set.
-func (ba *BatchRequest) IsNonKV() bool {
-	for _, union := range ba.Requests {
-		if (union.GetInner().flags() & isNonKV) == 0 {
-			return false
-		}
-	}
-	return true
-}
-
 // IsSingleSkipLeaseCheckRequest returns true iff the batch contains a single
 // request, and that request has the skipLeaseCheck flag set.
 func (ba *BatchRequest) IsSingleSkipLeaseCheckRequest() bool {

--- a/pkg/storage/replica.go
+++ b/pkg/storage/replica.go
@@ -2011,17 +2011,12 @@ func (r *Replica) executeReadOnlyBatch(
 		return nil, roachpb.NewError(err)
 	}
 
-	var endCmds *endCmds
-
-	if !ba.IsNonKV() {
-		// Add the read to the command queue to gate subsequent
-		// overlapping commands until this command completes.
-		log.Event(ctx, "command queue")
-		var err error
-		endCmds, err = r.beginCmds(ctx, &ba, spans)
-		if err != nil {
-			return nil, roachpb.NewError(err)
-		}
+	// Add the read to the command queue to gate subsequent
+	// overlapping commands until this command completes.
+	log.Event(ctx, "command queue")
+	endCmds, err := r.beginCmds(ctx, &ba, spans)
+	if err != nil {
+		return nil, roachpb.NewError(err)
 	}
 
 	log.Event(ctx, "waiting for read lock")
@@ -2033,9 +2028,7 @@ func (r *Replica) executeReadOnlyBatch(
 	// timestamp cache update is synchronized. This is wrapped to delay
 	// pErr evaluation to its value when returning.
 	defer func() {
-		if endCmds != nil {
-			endCmds.done(br, pErr, proposalNoRetry)
-		}
+		endCmds.done(br, pErr, proposalNoRetry)
 	}()
 
 	if err := r.IsDestroyed(); err != nil {
@@ -2167,29 +2160,24 @@ func (r *Replica) tryExecuteWriteBatch(
 		return nil, roachpb.NewError(err), proposalNoRetry
 	}
 
-	isNonKV := ba.IsNonKV()
-	var endCmds *endCmds
-	if !isNonKV {
-		// Add the write to the command queue to gate subsequent overlapping
-		// commands until this command completes. Note that this must be
-		// done before getting the max timestamp for the key(s), as
-		// timestamp cache is only updated after preceding commands have
-		// been run to successful completion.
-		log.Event(ctx, "command queue")
-		var err error
-		endCmds, err = r.beginCmds(ctx, &ba, spans)
-		if err != nil {
-			return nil, roachpb.NewError(err), proposalNoRetry
-		}
-
-		// Guarantee we remove the commands from the command queue. This is
-		// wrapped to delay pErr evaluation to its value when returning.
-		defer func() {
-			if endCmds != nil {
-				endCmds.done(br, pErr, retry)
-			}
-		}()
+	// Add the write to the command queue to gate subsequent overlapping
+	// commands until this command completes. Note that this must be
+	// done before getting the max timestamp for the key(s), as
+	// timestamp cache is only updated after preceding commands have
+	// been run to successful completion.
+	log.Event(ctx, "command queue")
+	endCmds, err := r.beginCmds(ctx, &ba, spans)
+	if err != nil {
+		return nil, roachpb.NewError(err), proposalNoRetry
 	}
+
+	// Guarantee we remove the commands from the command queue. This is
+	// wrapped to delay pErr evaluation to its value when returning.
+	defer func() {
+		if endCmds != nil {
+			endCmds.done(br, pErr, retry)
+		}
+	}()
 
 	var lease *roachpb.Lease
 	// For lease commands, use the provided previous lease for verification.
@@ -2205,31 +2193,29 @@ func (r *Replica) tryExecuteWriteBatch(
 		lease = status.lease
 	}
 
-	if !isNonKV {
-		// Examine the read and write timestamp caches for preceding
-		// commands which require this command to move its timestamp
-		// forward. Or, in the case of a transactional write, the txn
-		// timestamp and possible write-too-old bool.
-		if bumped, pErr := r.applyTimestampCache(&ba); pErr != nil {
-			return nil, pErr, proposalNoRetry
-		} else if bumped {
-			// If we bump the transaction's timestamp, we must absolutely
-			// tell the client in a response transaction (for otherwise it
-			// doesn't know about the incremented timestamp). Response
-			// transactions are set far away from this code, but at the time
-			// of writing, they always seem to be set. Since that is a
-			// likely target of future micro-optimization, this assertion is
-			// meant to protect against future correctness anomalies.
-			defer func() {
-				if br != nil && ba.Txn != nil && br.Txn == nil {
-					log.Fatalf(ctx, "assertion failed: transaction updated by "+
-						"timestamp cache, but transaction returned in response; "+
-						"updated timestamp would have been lost (recovered): "+
-						"%s in batch %s", ba.Txn, ba,
-					)
-				}
-			}()
-		}
+	// Examine the read and write timestamp caches for preceding
+	// commands which require this command to move its timestamp
+	// forward. Or, in the case of a transactional write, the txn
+	// timestamp and possible write-too-old bool.
+	if bumped, pErr := r.applyTimestampCache(&ba); pErr != nil {
+		return nil, pErr, proposalNoRetry
+	} else if bumped {
+		// If we bump the transaction's timestamp, we must absolutely
+		// tell the client in a response transaction (for otherwise it
+		// doesn't know about the incremented timestamp). Response
+		// transactions are set far away from this code, but at the time
+		// of writing, they always seem to be set. Since that is a
+		// likely target of future micro-optimization, this assertion is
+		// meant to protect against future correctness anomalies.
+		defer func() {
+			if br != nil && ba.Txn != nil && br.Txn == nil {
+				log.Fatalf(ctx, "assertion failed: transaction updated by "+
+					"timestamp cache, but transaction returned in response; "+
+					"updated timestamp would have been lost (recovered): "+
+					"%s in batch %s", ba.Txn, ba,
+				)
+			}
+		}()
 	}
 
 	log.Event(ctx, "raft")

